src/views/ExportSuccess.vue

```diff
@@ -26,12 +26,18 @@ import KeyguardClient from '@nimiq/keyguard-client';
 @Component({components: {SmallPage, Loader}})
 export default class ExportSuccess extends Vue {
     @Static private request!: ParsedSimpleRequest;
+    @Static private originalRouteName?: string;
     @State private keyguardResult!: KeyguardClient.ExportResult;
 
     private loaderState = Loader.State.LOADING;
     private successMessage = '';
 
     private async mounted() {
+        if (this.originalRouteName || this.request.kind === RequestType.MIGRATE) {
```

Comment:
Add a comment why we're not letting the `RpcApi` handle this as a side result.
And wouldn't one of the conditions be enough?